tests/ZendeskApi_v2.Tests/UserTests.cs

```diff
@@ -802,5 +802,38 @@ public async Task CanBulkUpdateUsers()
                 await Api.Users.DeleteUserAsync(r.Id);
             }
         }
+
+
+        public async Task CanBulkCreateUpdateUsersAsync()
+        {
+            var users = new List<User> {
+                new User
+                {
+                     Name = Guid.NewGuid().ToString("N") + " " + Guid.NewGuid().ToString("N"),
+                     Email = Guid.NewGuid().ToString("N") + "@test.com",
+                     Verified = true
+                },
+                new User
+                {
+                    Name = Guid.NewGuid().ToString("N") + " " + Guid.NewGuid().ToString("N"),
+                    Email = Guid.NewGuid().ToString("N") + "@test.com",
+                    Verified = true
+                },
+            };
+            var updateResp = await Api.Users.BulkCreateUpdateUsersAsync(users);
+            var job = await Api.JobStatuses.GetJobStatusAsync(updateResp.JobStatus.Id);
+            var count = 0;
+            while (job.JobStatus.Status.ToLower() != "completed" && count < 10)
+            {
+                await Task.Delay(1000);
+                job = await Api.JobStatuses.GetJobStatusAsync(updateResp.JobStatus.Id);
+                count++;
+            }
+            Assert.That(job.JobStatus.Status.ToLower(), Is.EqualTo("completed"));
+        }
+
     }
+
+
 }
+  
```

Comment:
- Fixed formatting issues (removed whitespace) 
- added required test attribute
- updated test case to cleanup test data. 

```suggestion

        [Test]
        public async Task CanBulkCreateUpdateUsersAsync()
        {
            var users = new List<User> {
                new User
                {
                     Name = Guid.NewGuid().ToString("N") + " " + Guid.NewGuid().ToString("N"),
                     Email = Guid.NewGuid().ToString("N") + "@test.com",
                     Verified = true
                },
                new User
                {
                    Name = Guid.NewGuid().ToString("N") + " " + Guid.NewGuid().ToString("N"),
                    Email = Guid.NewGuid().ToString("N") + "@tester.com",
                    Verified = true
                },
            };
            var updateResp = await Api.Users.BulkCreateUpdateUsersAsync(users);
            var job = await Api.JobStatuses.GetJobStatusAsync(updateResp.JobStatus.Id);
            var count = 0;
            while (job.JobStatus.Status.ToLower() != "completed" && count < 10)
            {
                await Task.Delay(1000);
                job = await Api.JobStatuses.GetJobStatusAsync(updateResp.JobStatus.Id);
                count++;
            }
            Assert.That(job.JobStatus.Status.ToLower(), Is.EqualTo("completed"));
            
            
            foreach(var u in users)
            {
                var rsp = await Api.Users.SearchByEmailAsync(u.Email);
                if(rsp.Users.Any())
                {
                   await Api.Users.DeleteUserAsync(rsp.Users[0].Id.Value);
                }
            }
        }
    }
}
```